python2/tests/test.bats

```diff
@@ -0,0 +1,17 @@
+TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
+
+@test "Version matches" {
+  result="$(hab pkg exec ${TEST_PKG_IDENT} python --version 2>&1 | head -1 | awk '{print $2}')"
+  [ "$result" = "${TEST_PKG_VERSION}" ]
+}
+
+@test "Help command" {
+  run hab pkg exec ${TEST_PKG_IDENT} python --help
+  [ "$status" -eq 0 ]
+}
+
+@test "Basic hello world" {
+  result="$(hab pkg exec ${TEST_PKG_IDENT} python ${BATS_TEST_DIRNAME}/test-hello-world.py)"
```

Comment:
Might be simpler to just do a `python -c "print('Hello World')"

Reply:
Perhaps. External files should grow over time as we add support for things like the SSL flags enabled in the build. Sticking with the file include for now.